### PR TITLE
enable windows binary module import

### PR DESCRIPTION
**Pull request by malyjacob**
enable wondows binary module importing, which allow no modification of the origin scripts using binary module importing in project.

try run on windows:
```
$ mkdir build
$ cd build
$ cmake .. -DBUILD_EXAMPLES=ON
$ cmake --build . --config Release
```

And after the above, try copy the target file `qjs.exe`, `fib.dll`, `point.dll` to dir `examples` and run the following:
```cmd
$ .\qjs.exe .\test_fib.js
$ .\qjs.exe .\test_point.js
```

And the output is all right.

### CODE REVIEW

**Comment by saghul** [comment]
Ops https://github.com/quickjs-ng/quickjs/pull/490 was in progress.

File: CMakeLists.txt [hunk 1]
```diff
@@ -292,12 +292,14 @@ if(BUILD_EXAMPLES AND NOT WIN32)
...
 
-    if(NOT WIN32)
+    # if(NOT WIN32)
         add_library(fib MODULE examples/fib.c)
         set_target_properties(fib PROPERTIES
             PREFIX ""
             C_VISIBILITY_PRESET default
         )
+        target_include_directories(fib PRIVATE .)
+        target_link_libraries(fib PRIVATE qjs)
```
Comment (saghul) [inline CMakeLists.txt:303]: is there no way to do lazy symbol binding on windows?

Comment (malyjacob) [reply CMakeLists.txt:303]: i am not sure

**Comment by saghul** [comment]
Done in https://github.com/quickjs-ng/quickjs/pull/490

**Pull request closed without merging**